library/library.mllib

```diff
@@ -1,5 +1,3 @@
-Univops
```

Comment:
I was wondering about this one. Guess I don't need to now.

Reply:
It is moved into engine, not removed. Maybe the natural step is to integrate it into some part of `engine`?

Reply:
I was precisely wondering why it was in library/.

Reply:
Due to a dependency problem I couldn’t put it in engine or pretyping (or somewhere in interp I believe), AFAIR